modules/olm-bundle/00_mod.mk

```diff
@@ -0,0 +1,52 @@
+# Utility functions
+fatal_if_undefined = $(if $(findstring undefined,$(origin $1)),$(error $1 is not set))
+first_existing_file = $(firstword $(wildcard $1))
+first_existing_file_or_default = $(firstword $(wildcard $1) $(firstword $1))
+
+# Utility variables
+current_makefile := $(lastword $(MAKEFILE_LIST))
+current_makefile_directory := $(dir $(current_makefile))
+olm_base_dir := $(current_makefile_directory)/base
+
+# Default config
+olm_publish_repos ?= 
+olm_project_id ?=
+oci_olm_image_tag ?= $(VERSION)
+preflight_container_project_ids ?=
+
+# The example directory is used to find CRD examples, this is used to create the
+# alm-examples annotation in the clusterserviceversion
+olm_example_directory ?= deploy/examples
```

Comment:
I wouldn't place the CRD examples in the deploy folder, since it relates to how to use the controller (instead of how to deploy it).
Maybe, we can also rename the folder to `cr_examples` for clarity.

Reply:
That would make sense, we also have eg. https://github.com/cert-manager/trust-manager/tree/main/docs/api docs which explain the API. We could maybe place those two together in the same dir?
Maybe `docs/examples`?